tools/bench-asyncio-write.py

```diff
@@ -74,7 +74,7 @@ def fm_time(s, _fms=("", "m", "µ", "n")):
 jobs = [
     (
         # always start with a 256B headers chunk
-        "{} / {}".format(fm_size(sum(j) if j else 0), len(j)),
+        f"{fm_size(sum(j) if j else 0)} / {len(j)}",
```

Comment:
Or maybe it's not pyupgrade (but, I don't see any other suspects)...

> note: pyupgrade is intentionally timid and will not create an f-string if it would make the expression longer or if the substitution parameters are anything but simple names or dotted names (as this can decrease readability).

So, it's either another thing, or pyupgrade is misbehaving.

Reply:
OK, pyupgrade documentation has been updated. This is expected behaviour, but a couple of these changes should ideally be simplified to make the code cleaner.